tools/sync_bugzilla_to_github.py

```diff
@@ -253,6 +254,9 @@ def sync_issue_from_bug_info(self, bugid, bug_info):
                     issue.number, changed_fields, **bug_info)
                 # Weird API thing where `issue.edit` accepts strings rather than label refs...
                 issue_info['labels'] = [l.name for l in issue_info['labels']]
+                # Explain why we are closing this issue.
+                if not bug_info['is_open'] and 'resolution' in bug_info:
```

Comment:
nit: please include `'state' in changed_fields` in this `if` condition here, so it reads "if we are changing the issue status, and the bug is not open in bugzilla, and we have the resolution from bugzilla, then...".